### DISCUSSION TITLE
slash in collection name

### DISCUSSION THREAD

**Discussion by Tchad90** [dani-garcia/vaultwarden#3530] [Q&A]
Hello,

We are on the Version 2023.3.0 of vaultwarden. And when my users create collections. This one is systematically prefixed with a '/' (example below).

![2023-05-15_14h47_21](https://github.com/dani-garcia/vaultwarden/assets/85280463/c5ab7a7e-6e33-4f90-88b4-fc33b42c5226)

how can I correct this please?  This is a collection imported from the 'Bitwarden (CSV)' source

**Comment by BlackDex** [comment]
First, it's not something Vaultwarden can fix. This is all client side handled. Vaultwarden doesn't even receive the plain text version of the collection names.

It could be a bug in the client code.
I suggest to test it on vault.bitwarden.com and if there it's also an issue, report it upstream, using the SAAS solution as an example.
